tests/reg_issue270/issue270.fz

```diff
@@ -0,0 +1,4 @@
+ex
+  pre false
```

Comment:
The idea of a negative test is that it contains numbered lines with comment `# 1. should flag an error: ...this or that should be detected...` and then negative.mk checks that they all are actually reported.

Reply:
@fridis I did not add this since the precondition failure occurs at run time.
I added a `should flag an error` comment now and a `skip_c` file. The minor downside to is that now it does not check anymore if the compiled program also terminates erroneously.